markdown/solace-pubsub-microgateway/solace-pubsub-microgateway.md

```diff
@@ -0,0 +1,1185 @@
+author: Aaron Lee
+summary: Summary of your codelab that is human readable
+id: solace-pubsub-microgateway
+tags: iguide
+categories: Java,REST,API,pub/sub
```

Comment:
So a bit weird, but the first category listed is what is aded to the "Filter by category" bar on codelabs.solace.dev. It is also what decides the style of the codelab on the landing page. I think for these design pattern type ones that use solace features we should put "Solace" first so it gets the Solace logo rather than the java logo or rest logo.

Reply:
Wouldn't they ALL be Solace ones?  Question is: if these are only on our codelabs.solace.dev landing page, I think this one should have REST.  But if these are for listing on a public (Google?) master site, then maybe Solace should be first.

Reply:
Okay, I can get behind REST. That makes sense. Let's go with that as the first one. So maybe "REST,Solace,Java"

Honestly as I type this I'm kind of wondering if it makes sense to disable the "Filter by category" until we have way more. I believe we can do that and keep the "Filter by type"

Reply:
Agreed!  Closing.